plugins/azure/virtualnetworks/ddosStandardProtectionEnabled.spec.js

```diff
@@ -0,0 +1,93 @@
+var expect = require('chai').expect;
+var ddosStandardProtectionEnabled = require('./ddosStandardProtectionEnabled');
+
+const virtualNetworks = [
+    {
+        name: 'test-vnet',
+        id: '/subscriptions/123/resourceGroups/aqua-resource-group/providers/Microsoft.Network/virtualNetworks/test-vnet',
+        type: 'Microsoft.Network/virtualNetworks',
+        location: 'eastus',
+        provisioningState: 'Succeeded',
+        virtualNetworkPeerings: [],
+        enableDdosProtection: true
+    },
+    {
+        name: 'test-vnet',
+        id: '/subscriptions/123/resourceGroups/aqua-resource-group/providers/Microsoft.Network/virtualNetworks/test-vnet',
+        type: 'Microsoft.Network/virtualNetworks',
+        location: 'eastus',
+        provisioningState: 'Succeeded',
+        virtualNetworkPeerings: [],
+        enableDdosProtection: false
+    }
+];
```

Comment:
Use double quotes notation to define objects
#586

Reply:
Done as suggested.